pages/Javascript-API.md

```diff
@@ -56,13 +89,69 @@ The properties of the feature at the current cursor location may be accessed thr
 
 This is accomplished by assigning a unique color to each feature onscreen and rendering the scene to an offscreen buffer. When queried, the `scene.selection` object checks the offscreen render at the current cursor position, and identifies the feature by its color.
 
-#### `requestRedraw()`
-Requests an update to the drawn map. If `animated: true` is set, this happens once per frame automatically.
-
 #### `getActiveCamera()`
+Returns the active camera.
 
-#### `setActiveCamera()`
+#### `requestRedraw()`
+Requests an update to the drawn map. If `animated: true` is set, this happens once per frame automatically.
 
 #### `reload(scene_url)`
```

Comment:
`reload()` is still active for backwards but essentially deprecated, we should use `load()` (same arguments) instead. There is also a second, rarely needed argument for `load()`, which is an explicit scene *base path* to use (rather than inferring the base path from the scene URL, which is the default behavior). The base path will be prefixed to paths for resources relative to the scene file, such as images. The primary use for this argument right now is for Tangram Play, where a local blob URL is passed in as the scene content, but we want resources in the scene to reference the original base path from which the scene was loaded.